lua/lazyvim/config/options.lua

```diff
@@ -4,6 +4,13 @@ vim.g.maplocalleader = "\\"
 
 -- LazyVim auto format
 vim.g.autoformat = true
+-- LazyVim auto format mode (files | changes)
```

Comment:
```suggestion
-- LazyVim auto format mode (file | changes)
```
Just a small typo so users don't get confused.